### DISCUSSION TITLE
Do we need both flake8 and black?

### DISCUSSION THREAD

**Discussion by m4rc1e** [fonttools/fontbakery#4438] [General]
I <3 black and despise flake8. Can anyone convince me that flake8 actually provides any value? What is starting to infuriate me is that Black will suggest a fix and then flake8 will count it as a fail.

If I had my way, I'd shoot flake8 into the sun.

**Comment by anthrotype** [comment]
cool kids nowadays are moving to https://github.com/astral-sh/ruff

**Comment by felipesanches** [comment]
@m4rc1e, We've just successfully launched `flake8` into a collision trajectory towards the Sun.

https://github.com/fonttools/fontbakery/pull/4439
